src/transformStateWithClones.js

```diff
@@ -6,8 +6,76 @@
  *
  * @return {Object[]}
  */
-function transformStateWithClones(state, actions) {
-  // write code here
+
+const state = {
+  foo: 'bar',
+  bar: 'foo',
+};
+
+const actions = [
+  {
+    type: 'addProperties',
+    extraData: { name: 'Jim', hello: 'world' },
+  },
+  {
+    type: 'removeProperties',
+    keysToRemove: ['bar', 'hello'],
+  },
+  {
+    type: 'addProperties',
+    extraData: { another: 'one' },
+  },
+  {
+    type: 'clear',
+  },
+];
+
+function transformStateWithClones(initState, objActions) {
+  const ArrState = [];
```

Comment:
```suggestion
  const arrState = [];
```
should be in camelCase